pkg/api/volumes.go

```diff
@@ -280,6 +308,24 @@ func (client *Client) chooseLUN(initiators []string) (int, error) {
 
 	sort.Sort(Volumes(allvolumes))
 
+	if volumeID != "" {
+		lun := -1
+		for _, vol := range allvolumes {
+			if vol.Name == volumeID {
+				if lun < 1 {
+					lun = vol.LUN
+				} else if lun != vol.LUN {
+					return -1, fmt.Errorf("found multiple LUNs (%d, %d) for volume %s", lun, vol.LUN, volumeID)
```

Comment:
Why is this an error? Is this restriction in line with other user interfaces, or just an assumption we want to make to keep the API code simpler?